packages/appstore/src/components/loader/devider/devider.scss

```
@@ -0,0 +1,11 @@
+.vertical-devider {
```

just a nit there is a spelling error here.
```suggestion
.vertical-divider {
```